src/MOIWrapper.jl

```diff
@@ -0,0 +1,283 @@
+LQOI.backend_type(m::CplexOptimizer, ::MOI.Nonpositives)         = Cchar('L')
+LQOI.backend_type(m::CplexOptimizer, ::MOI.Nonnegatives)         = Cchar('E')
+
+# TODO - improve single type
+function LQOI.change_variable_bounds!(instance::CplexOptimizer, 
+        colvec::Vector{Int}, valvec::Vector{Float64}, sensevec::Vector{Cchar})
+    
+    chgbds(instance.inner, ivec(colvec .- 1), sensevec, valvec)
+end
+
+function LQOI.get_variable_lowerbound(instance::CplexOptimizer, column::Int)
+    get_varLB(instance.inner, Cint(column))[1]
+end
+
+function LQOI.get_variable_upperbound(instance::CplexOptimizer, column::Int)
+    get_varUB(instance.inner, Cint(column))[1]
+end
+
+function LQOI.get_number_linear_constraints(instance::CplexOptimizer)
+    num_constr(instance.inner)
+end
+
+function LQOI.add_linear_constraints!(instance::CplexOptimizer,
+        A::LQOI.CSRMatrix{Float64}, sense::Vector{Cchar}, rhs::Vector{Float64})
+
+    add_rows!(instance.inner, ivec(A.row_pointers), ivec(A.columns), 
+              A.coefficients, sense, rhs)
+end
+
+function LQOI.get_rhs(instance::CplexOptimizer, row::Int)
+    get_rhs(instance.inner, Cint(row))
+end
+
+function LQOI.get_linear_constraint(instance::CplexOptimizer, row::Int)
+    (nzcnt, rmatbeg, rmatind, rmatval) = 
+            get_rows(instance.inner, Cint(row), Cint(row))
+    return rmatind[1:nzcnt] .+ 1, rmatval[1:nzcnt]
+end
+
+function LQOI.change_matrix_coefficient!(instance::CplexOptimizer, 
+                                         row::Int, col::Int, coef::Float64)
+                                         
+    chg_coef!(instance.inner, Cint(row), Cint(col), coef)
+end
+
+function LQOI.change_objective_coefficient!(instance::CplexOptimizer, col::Int, 
+                                            coef::Float64)
+                                            
+    set_obj!(instance.inner, [Cint(col)], [coef])
+end
+
+function LQOI.change_rhs_coefficient!(instance::CplexOptimizer, row::Int, 
+                                      coef::Float64)
+                                      
+    chg_rhs(instance.inner, [Cint(row)], [coef])
+end
+
+function LQOI.delete_linear_constraints!(instance::CplexOptimizer, 
+                                         first_row::Int, last_row::Int)
+                                         
+    del_rows!(instance.inner, Cint(first_row), Cint(last_row))
+end
+
+function LQOI.change_variable_types!(instance::CplexOptimizer, 
+        columns::Vector{Int}, vtypes::Vector{Cchar})
+    
+    chg_ctype!(instance.inner, ivec(columns), vtypes)
+end
+
+function LQOI.change_linear_constraint_sense!(instance::CplexOptimizer, 
+        rows::Vector{Int}, senses::Vector{Cchar})
+                                              
+    chg_sense!(ivec(rows), senses)
+end
+
+function LQOI.set_linear_objective!(instance::CplexOptimizer, 
+        columns::Vector{Int}, coefficients::Vector{Float64})
+    
+    n = num_var(instance.inner)    
```

Comment:
Style: no one-letter variable names

Reply:
But `n` and `m` are often used in linear programming. And these are local vars in functions that always fully fit in a laptop screen; so if one doesn't know what the variable is, he can just look above.